src/id.mjs

```diff
@@ -54,7 +54,7 @@ export function toDigest(value) {
   const copy = canonicalize({ ...value });
   const canonical = encode(copy);
   const digest = toHex(keccak256(canonical));
-  const index = `${value.timestamp.toString(16)}${digest}`;
+  const index = `${value.timestamp.toString(16).padStart(8, "0")}${digest}`;
```

Comment:
This shouldn't affect any existing messages, but ensures that the ordering is correct if we somehow get a very small timestamp value.

Reply:
I think there cannot ever be a timestamp that is smaller than

https://github.com/attestate/kiwistand/blob/main/src/store.mjs#L263 MIN_TIMESTAMP_SECS=1672527600 within the network, unless someone attempts to fork away from network consensus rules